iroha/src/kura.rs

```diff
@@ -106,6 +101,18 @@ impl Kura {
             .map(|block| block.header.height)
             .unwrap_or(0)
     }
+
+    pub fn blocks_starting_at_hash(&self, hash: Hash) -> Option<&[ValidBlock]> {
```

Comment:
I think it will be enough to name the method `blocks_after` and the full signature will be descriptive enough: `blocks_after(hash: Hash)`?